App/Actions/AppActions.js

```diff
@@ -50,4 +50,47 @@ module.exports = {
     });
   },
 
+  // User
+
+  loadUser() {
+    StorageWrapper.getUser()
+    .then((value) => {
+
+      var user = null;
+      if (value !== null) {
+        user = JSON.parse(value);
+      }
+
+      dispatcher.handleViewAction({
+        actionType: AppConstants.LOAD_USER,
+        data: user,
+      });
+    })
+  },
+
+  skipLogin() {
+    StorageWrapper.setUser('Anonymous', null)
+    .then(function() {
```

Comment:
Just a comment, we should be using ES6 features as much as we can, since they are better practice. This should be written in the arrow function syntax. `() => { ... }`